inttest/Makefile

```diff
@@ -47,6 +47,9 @@ check-customports: TIMEOUT=6m
 # Config change smoke runs actually many cases hence a bit longer timeout
 check-configchange: TIMEOUT=8m
 
+# Node role check runs several cases
+check-noderole: TIMEOUT=6m
```

Comment:
why to have extended timeout?

Reply:
There are 3 test cases and 4 minutes is not enough.